packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -344,25 +412,55 @@ export const setupRDSClusterAndData = async (config: RDSConfig, queries?: string
         resourceArn: dbCluster.clusterArn,
         secretArn: dbCluster.secretArn,
         sql: query,
-        database: sanitizedDbName,
+        database: config.dbname,
       };
       const executeStatementResponse = await client.send(new ExecuteStatementCommand(executeStatementInput));
-      console.log('Create table response: ' + JSON.stringify(executeStatementResponse));
+      console.log('Run query response: ' + JSON.stringify(executeStatementResponse));
     } catch (err) {
       throw new Error(`Error in creating tables in test database: ${JSON.stringify(err, null, 4)}`);
     }
   }
 
+  /*try {
+    const executeStatementInput: ExecuteStatementCommandInput = {
+      resourceArn: dbCluster.clusterArn,
+      secretArn: dbCluster.secretArn,
+      sql: `drop database ${config.dbname} with (FORCE)`,
+      database: dbCluster.dbName,
+    };
+    const executeStatementResponse = await client.send(new ExecuteStatementCommand(executeStatementInput));
+    console.log('Run query response: ' + JSON.stringify(executeStatementResponse));
+  } catch (err) {
+    throw new Error(`Error in running queries in test database: ${err}`);
+  }*/
+
```

Comment:
Unless there's a pretty compelling reason to retain this (e.g., you're going to reinstate it soon), please remove dead code so it doesn't clutter up the codebase.